src/libical/icaldate_p.h

```diff
@@ -1,6 +1,22 @@
+/**
+  SPDX-FileCopyrightText: 1986-2000, Hiram Clawson
+  SPDX-FileCopyrightText: 2018, Markus Minichmayr <markus@tapkey.com>
+  SPDX-License-Identifier: LGPL-2.1-only OR MPL-2.0
+*/
+
 /*
- * This work is based on work from Hiram Clawson and has been modified to the
- * needs of the libical project. The original copyright notice is as follows:
+ * WARNING: USE AT YOUR OWN RISK
```

Comment:
Does this make it into ical.h? Even if it does, people might miss it. Maybe add a similar comment (maybe even shorter) to each individual internal function?